example/package-lock.json

```diff
@@ -24,6 +24,7 @@
       }
     },
     "..": {
+      "name": "oscal-react-library",
```

Comment:
Why is this being added with no corresponding change to `package.json`?

Reply:
I'm not really sure why, I've never manually touched either file.

Reply:
Let's see if we can find that out. I am concerned that we're going to keep seeing PRs battle over changing this line in this file.

Reply:
Wasn't this issue looked into a bit before with #132 ?